src/index.html

```diff
@@ -1,12 +1,390 @@
+  <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+    rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="header">
+        <a
+          href="#home"
+          class="logo">
+          <img
+          src="images/logo.png"
+          alt="logo">
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+              href="#apple"
+              class="nav__link is-active">Apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+              href="#samsung"
+              class="nav__link">Samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+              href="#smartphones"
+              class="nav__link">Smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#laptops"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+              href="#gadgets"
+              class="nav__link">Gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+              href="#tablets"
+              class="nav__link">Tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+              href="#photo"
+              class="nav__link">Photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+              href="#video"
+              class="nav__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <main class="main" >
+        <article class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+
+          <h3 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+
+          <p class="card__code">Product code: 195434
+          </p>
+
+          <div class="stars-block">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="stars-block__review">Reviews: 5</p>
```

Comment:
```suggestion
          <div class="card__stars">
            <div class="stars">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>

            <p class="card__stars-review">Reviews: 5</p>
```
There should be only 2 blocks: card and stars. All other are just elements